src/services/FileUploadService.ts

```diff
@@ -0,0 +1,54 @@
+import axios, { AxiosResponse } from "axios";
+import multer from "multer";
+import express from "express";
+import { S3 } from "aws-sdk";
+import fs from 'fs';
+
+import config from "../config";
+import { validateFileUpload } from "../validators/FileUploadValidator";
+import { error } from "console";
+import { randomUUID } from "crypto";
+
+
+export const uploadToS3 = async (s3: S3, fileData?: Express.Multer.File) => {
+  try {
+
+    if (!fileData) {
+      throw new Error('No file data provided');
+    }
+    const filePath = fileData.path;
+    const fileContent = fs.readFileSync(filePath);
+    const guid = randomUUID();
+    const folderData = "the_extra_semicolon/imports/"+ guid + fileData!.originalname;
+    console.log(folderData);
+
+
+      const params = {
+        Bucket: config.BUCKET_NAME || 'academy-job-portal-cvs',
```

Comment:
The default value that you are setting here ('academy-job-portal-cvs') should be done in the config file

Reply:
It is in config but for some reason it was not picking it up